scout/commands/base.py

```diff
@@ -53,6 +52,8 @@ def get_app(ctx=None):
         )
         with open(options.params["config"], "r") as in_handle:
             cli_config = yaml.load(in_handle, Loader=yaml.SafeLoader)
+            if cli_config.get("mongodb"):
+                cli_config["MONGO_DBNAME"] = cli_config.pop("mongodb")
```

Comment:
Yes, that looks like it would work! You are going to hate me, but wouldn't it logically be better here:
https://github.com/Clinical-Genomics/scout/blob/8c8373150f2a68bb1f9ca5fa46409111a79c6eca/scout/utils/config.py#L33-L32
😊

But I guess I would have bought 
```suggestion
                cli_config["mongo_dbname"] = cli_config.pop("mongodb")
```
here. And the upper version isn't that long a step. But trust me when I say that will look confusing next time we have to think about it.

Reply:
> Yes, that looks like it would work! You are going to hate me, but wouldn't it logically be better here:
> https://github.com/Clinical-Genomics/scout/blob/8c8373150f2a68bb1f9ca5fa46409111a79c6eca/scout/utils/config.py#L33-L32
> 😊

I don't hate anybody, especially for work-related stuff 😆 

I considered what you are suggesting, but then saw this code a few lines down and figured the new code was consistent with the old, this way.

<img width="884" height="708" alt="image" src="https://github.com/user-attachments/assets/e7eaba99-c67f-4b16-acf9-cad7e167bdc4" />


This ways utils/config.py stays cleaner also